<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubcavityPlugin</class>
 <widget class="QWidget" name="SubcavityPlugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>326</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>326</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Subpocket Chacterization</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>931</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <widget class="QWidget" name="leftpart" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>320</width>
         <height>237</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>237</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_input_subcavs">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <underline>true</underline>
             </font>
            </property>
            <property name="text">
             <string>Input options</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_cavID_subcavs">
              <property name="text">
               <string>Cavity ID to decompose...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="cavID"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="decomposeall">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>... or decompose all cavities</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_outputpymol_subcavs">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <underline>true</underline>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>PyMOL and .pml file options</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="pymol_subcavs">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>Open PyMOL when the job is over</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="bysubcav">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>Coloring by subcavity by default</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="run_subcavs">
            <property name="text">
             <string notr="true">Run</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_close_subcavs">
            <property name="text">
             <string notr="true">Close</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>0</height>
       </size>
      </property>
      <widget class="QLabel" name="label_resultslog_subcav">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>580</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Results and log</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QTextBrowser" name="output_subcavs">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>591</width>
         <height>191</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
        </font>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="title_subcav">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>926</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Subcavity decomposition panel.&lt;/p&gt;&lt;p&gt;Cavities are decomposed into subcavities based on geometry.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>961</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
